libretro/libretro-mapper.c

```diff
@@ -55,12 +55,12 @@ extern bool real_ntsc;
 
 int vkey_pos_x = 0;
 int vkey_pos_y = 0;
-int vkflag[8]={0};
-static int jflag[4][16]={0};
-static int kjflag[2][16]={0};
-static int mflag[2][16]={0};
-static int jbt[2][24]={0};
-static int kbt[16]={0};
+int vkflag[8] = {0};
+int jflag[4][16] = {0};
+int kjflag[2][16] = {0};
+int mflag[2][16] = {0};
+int jbt[2][24] = {0};
+int kbt[16] = {0};
```

Comment:
Are these accessed anywhere else? Looks like they should stay `static` to me.

Reply:
Good point, changed back.